yt_dlp/extractor/nebula.py

```diff
@@ -252,7 +317,7 @@ class NebulaCollectionIE(NebulaBaseIE):
                 'title': 'Lindsay Ellis',
                 'description': 'Enjoy these hottest of takes on Disney, Transformers, and Musicals.',
             },
-            'playlist_mincount': 100,
+            'playlist_mincount': 10,
```

Comment:
wasn't the mincount set high to test pagination? What happened?

Reply:
Yeah, this was not a well thought-out change.

Originally, the number was arbitrarily set. It wasn't even to test pagination (wasn't it back then anyway requesting all pages?), I just did not know what the consequences are. Now I noticed that this causes the test cases to request all 100 videos, and I thought I could speed them up (barely!) by reducing this number.

I changed it back. I guess testing pagination makes sense here and the extra second(s) the test takes don't really matter.